timescale-container/test/ql-db-init/1-mtlusovini.etdevice-ddl.sql

```diff
@@ -0,0 +1,35 @@
+CREATE SCHEMA IF NOT EXISTS mtlusovini;
```

Comment:
i am not sure i would refer to lusovini in a public repo.

Reply:
got rid of it, thx!